plugins/out_azure_kusto/azure_kusto.c

```diff
@@ -219,6 +227,13 @@ static int cb_azure_kusto_init(struct flb_output_instance *ins, struct flb_confi
         return -1;
     }
 
+    const char *tmp = flb_output_get_property("ingestion_endpoint_connect_timeout", ins);
```

Comment:
I think this is not necessary, because the config map defaults to ` FLB_AZURE_KUSTO_INGEST_ENDPOINT_CONNECTION_TIMEOUT` (btw, let's fix that name)